language_formatters_pre_commit_hooks/pretty_format_java.py

```diff
@@ -50,22 +51,71 @@ def get_urls(_version: str) -> typing.List[str]:
         )
 
 
+def _download_palantir_java_formatter_jar(version: str) -> str:  # pragma: no cover
+    def get_urls(_version: str) -> typing.List[str]:
+        # Links extracted from https://github.com/jsonschema2dataclass/palantir-cli
+        return [
+            "https://github.com/jsonschema2dataclass/palantir-cli/releases/download/"
+            "{version}/palantir-cli-{version}-standalone.jar".format(
+                version=_version,
+            ),
+        ]
+
+    possible_urls = get_urls(version)
```

Comment:
I know this is copied from other parts of the code but IMO I would inline this function